diff -ruN original/refine.ini patched/refine.ini
--- original/refine.ini	2013-08-27 12:55:52.000000000 -0300
+++ patched/refine.ini	2015-07-30 16:17:38.843049172 -0300
@@ -5,7 +5,7 @@
 #REFINE_PORT=3334
 #REFINE_HOST=127.0.0.1
 #REFINE_WEBAPP=main\webapp
-REFINE_MEMORY=1400M
+#REFINE_MEMORY=1400M
 
 # Some sample configurations. These have no defaults.
 #ANT_HOME=C:\grefine\tools\apache-ant-1.8.1
